include/old/test.h

```diff
@@ -44,6 +44,9 @@
 #include <string.h>
 #include <stdlib.h>
 #include <stdint.h>
+#ifndef __GLIBC__
+#include <bits/reg.h>
+#endif
 
```

Comment:
yeah, we don't want this.

we can't use C code to define this as other places expand it in CPP context.  i think we can just write something like:

```
#if __WORDSIZE == 64
# define LTP_USE_64_ABI
#else
... existing logic ...
#endif
```